deps/exokit-bindings/webaudiocontext/src/AudioContext.cpp

```diff
@@ -427,14 +427,14 @@ NAN_METHOD(AudioContext::CreateBufferSource) {
 NAN_METHOD(AudioContext::CreateScriptProcessor) {
   Nan::HandleScope scope;
 
-  uint32_t bufferSize = info[0]->IsNumber() ? info[0]->Uint32Value() : 256;
-  uint32_t numberOfInputChannels = info[1]->IsNumber() ? info[1]->Uint32Value() : 2;
-  uint32_t numberOfOutputChannels = info[2]->IsNumber() ? info[2]->Uint32Value() : 2;
+  uint32_t bufferSize = info[0]->IsNumber() ? TO_UINT32(info[0]) : 256;
```

Comment:
Should `info.Length() >= 3` be checked here?

Reply:
I think the number extraction is sufficient for now. Guarding all cases against invalid arguments would be another holistic effort in these APIs.